src/modules/commentsModal.js

```diff
@@ -122,7 +128,41 @@ const generateModal = (id) => {
         li.appendChild(liText);
         ulComments.appendChild(li);
       });
-      // <li>Luis on Aug 30: Comment 3</li>
+    }
+  });
+
+  // Event listener for the submit button
+  submitBtn.addEventListener('click', () => {
+    if (nameInput.value === '' || commentTextArea.value === '') {
+      nameInput.classList.add('empty-input');
+      commentTextArea.classList.add('empty-input');
+      setTimeout(() => {
+        nameInput.classList.remove('empty-input');
+        commentTextArea.classList.remove('empty-input');
+      }, 1000);
+    } else {
+      const nameValue = nameInput.value;
+      const commentValue = commentTextArea.value;
+      nameInput.value = '';
+      commentTextArea.value = '';
+      const promise = addComment(nameValue, commentValue, id);
+      promise.then((response) => {
+        if (response.status === 201) {
+          ulComments.innerHTML = '';
+          const promiseComments = getComments(id);
+
+          promiseComments.then((comments) => {
+            comments.forEach((comment) => {
+              const li = document.createElement('li');
+              const liText = document.createTextNode(
+                `${comment.username} on ${comment.creation_date}: ${comment.comment}`,
+              );
+              li.appendChild(liText);
+              ulComments.appendChild(li);
+            });
+          });
+        }
+      });
     }
   });
 };
```

Comment:
# Everything looks awesome my friend
- I'm just looking that here you have a lot of nested code inside a if statements, you can avoid that using return in each statement

Reply:
Thank you for your time! I will take a look into that.